api/jiekouapi.php

```diff
@@ -16,9 +18,9 @@
     echo '<?xml version="1.0" encoding="UTF-8"?>';
     echo "\n";
     date_default_timezone_set("Asia/Shanghai");
-    $con=mysql_connect("localhost","root","19951025");
-    mysql_select_db("capubbs",$con);
-    mysql_query("set names 'utf8'",$con);
+    dbconnect;
+    mysql_select_db("capubbs");
+    mysql_query("set names 'utf8'");
```

Comment:
I wonder whether it is possible to make
```
mysql_select_db("capubbs");
mysql_query("set names 'utf8'");
```
inside of `dbconnect` so we don't need to specify it every time.